resource-managers/mesos/src/main/scala/org/apache/spark/deploy/mesos/ui/MesosClusterPage.scala

```diff
@@ -99,6 +103,12 @@ private[mesos] class MesosClusterPage(parent: MesosClusterUI) extends WebUIPage(
     <tr>
       <td><a href={s"driver?id=$id"}>{id}</a></td>
       {historyCol}
+      <td>
+      {
+      state.driverDescription.conf
+        .get("spark.mesos.dispatcher.queue", config.DISPATCHER_QUEUE.defaultValueString)
+    }
```

Comment:
indentation?

Reply:
You can revise like https://github.com/apache/spark/pull/30352/files#r524479806 .

Reply:
Formatted :heavy_check_mark: